protocols/cosi/packets.go

```diff
@@ -44,8 +44,7 @@ type ProtocolPacket struct {
 // ProtocolIO implements the sda.ProtocolIO interface for the CoSi protocol.
 type ProtocolIO struct{}
 
-// Wrap implements the sda.ProtocolIO interface by wrapping up any of the
-// four-step messages into a ProtooclPacket.
+// Wrap takes a dynamic OverlayMessage and returns a ProtocolPacket.
```

Comment:
what is a "dynamic" message ? I would not put dynamic unless we have a clear definition of what it is exactly. Same for the other occurences.